README.md

```diff
@@ -26,138 +23,131 @@ Kubernetes is:
-
-### Code of conduct
-
-Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
-
-### Are you ready to add to the discussion?
-
-We have presence on:
+Join us on social media and read our blog:
 
  * [Twitter](https://twitter.com/kubernetesio)
  * [Google+](https://plus.google.com/u/0/b/116512812300813784482/116512812300813784482)
- * [Blogger](http://blog.kubernetes.io/)
-
-You can also view recordings of past events and presentations on our [Media page](http://kubernetes.io/media/).
+ * [Blog](http://blog.kubernetes.io/)
 
-For Q&A, our threads are at:
+Ask questions and help answer them on:
 
  * [Stack Overflow](http://stackoverflow.com/questions/tagged/kubernetes)
  * [Slack](http://slack.k8s.io/)
 
-### Want to contribute to Kubernetes?
+Attend our key events:
 
-If you're interested in being a contributor and want to get involved in developing Kubernetes, start in the [Kubernetes Developer Guide](docs/devel/README.md) and also review the [contributor guidelines](CONTRIBUTING.md).
+* [kubecon](http://events.linuxfoundation.org/events/kubecon)
+* [cloudnativecon](http://events.linuxfoundation.org/events/cloudnativecon)
+* weekly [community meeting](https://github.com/kubernetes/community/blob/master/community/README.md)
 
-Or, if you just have an idea for a new feature, see the [Kubernetes Features](https://github.com/kubernetes/features) repository for details on how to propose it.
+Join a Special Interest Group ([SIG](https://github.com/kubernetes/community))
 
-Also, please see our [expectations](docs/devel/community-expectations.md) for members of
-the Kubernetes community.
+## Contribute
 
-### Support
+If you're interested in being a contributor and want to get involved in developing Kubernetes, get started with this light reading:
 
-While there are many different channels that you can use to get ahold of us, you can help make sure that we are efficient in getting you the help that you need.
+*  The community [expectations](docs/devel/community-expectations.md)
+*  The [contributor guidelines](CONTRIBUTING.md)
+*  The [Kubernetes Developer Guide](docs/devel/README.md)
 
-If you need support, start with the [troubleshooting guide](docs/troubleshooting.md#getting-help) and work your way through the process that we've outlined.
+You will then most certainly gain a lot from joining a [SIG](https://github.com/kubernetes/community), attending the regular hangouts as well as the community [meeting](https://github.com/kubernetes/community/blob/master/community/README.md).
 
-That said, if you have questions, reach out to us one way or another.  We don't bite!
+If you have an idea for a new feature, see the [Kubernetes Features](https://github.com/kubernetes/features) repository for a list of features that are coming in new releases as well as details on how to propose one.
 
-### Community resources
+## Documentation
```

Comment:
There's a link to the doc site above, so I think we can just remove this section.